### PR TITLE
AOAF-340: Added an error 404 page and corrected the file more package…

### PR DESCRIPTION
**Pull request by okoye-companieshouse**
… link

### CODE REVIEW

File: src/views/partials/error_400.njk [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{% extends "layouts/shell.njk" %}
+
+{% block main_content %}
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl govuk-!-margin-bottom-8">
+            Page not found
+        </h1>
+        <pre class="govuk-!-margin-top-8">Status code: 404</pre>
```
[the remark this answers is not in the document]
Comment (okoye-companieshouse) [reply src/views/partials/error_400.njk:8]: Yes that is fine

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/views/partials/error_400.njk (hunk 1 above)
[the remark this answers is not in the document]
Comment (okoye-companieshouse) [reply src/views/partials/error_400.njk]: Sorted changed the message to match the name

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**